tldr/hybrid_extractor.py

```
@@ -342,6 +351,14 @@ def extract(self, file_path: str | Path, base_path: str | None = None) -> Module
                     return result
             logger.debug(f"Tree-sitter-elixir not available or failed, using Pygments for {suffix}")
 
+        # Zig - use tree-sitter-zig if available
+        if suffix in self.ZIG_EXTENSIONS:
+            if TREE_SITTER_ZIG_AVAILABLE:
+                result = self._try_tree_sitter(self._extract_zig, file_path, "zig")
+                if result:
+                    return result
+            logger.debug(f"Tree-sitter-zig not available or failed, using Pygments for {suffix}")
```

Ruff (0.14.14)

Logging statement uses f-string

(G004)